new.css

```diff
@@ -213,14 +214,14 @@ input[type="reset"],
 input[type="button"] {
 	font-size: 1rem;
 	display: inline-block;
-	padding: 6px 12px;
+	padding: .5rem .75rem; /* updated pixel values to rem */
```

Comment:
Why not `0.375rem .75rem` to match previous? It looks different w/ too much vertical padding as defined.

Reply:
That's correct, my bad will update this.